lib/views/widgets/Events/event_card.dart

```diff
@@ -0,0 +1,224 @@
+                          child: Text(
+                            widget.event.clusterName,
+                            style: TextStyle(
+                                fontWeight: FontWeight.bold,
+                                fontSize: totalHeight * 0.016,
+                                fontFamily:
+                                    GoogleFonts.montserrat().fontFamily),
+                          ),
+                        ),
+                      ),
+                      // Expanded(
+                      //   flex: ,
+                      //   child: Container(
+                      //   ),
+                      // ),
+                      Expanded(
+                        flex: 70,
+                        child: Center(
+                          child: Text(
+                            widget.event.desc,
+                            style: TextStyle(
+                                fontWeight: FontWeight.bold,
+                                fontSize: totalHeight * 0.011,
+                                fontFamily:
+                                    GoogleFonts.montserrat().fontFamily),
+                          ),
+                        ),
+                      ),
+                      // Expanded(
+                      //   child: SizedBox(
+                      //     height: 45,
+                      //   ),
+                      // ),
+                      Expanded(
+                        flex: 25,
+                        child: Center(
+                          child: Text(
+                            widget.event.cupName,
+                            style: TextStyle(
+                                fontWeight: FontWeight.bold,
+                                fontSize: totalHeight * 0.0184,
+                                fontFamily:
+                                    GoogleFonts.montserrat().fontFamily),
+                          ),
+                        ),
+                      ),
+                      Expanded(
+                        flex: 25,
+                        child: Center(
+                          child: Text(
+                            'click here for rulebook -->',
+                            style: TextStyle(
+                                fontWeight: FontWeight.bold,
+                                fontSize: totalHeight * 0.0162,
+                                fontFamily:
+                                    GoogleFonts.montserrat().fontFamily),
+                          ),
+                        ),
+                      ),
+                      // Expanded(
+                      //   child: SizedBox(
+                      //     height: 25,
+                      //   ),
+                      // ),
+                      Expanded(
+                        flex: 25,
+                        child: Center(
+                          child: Text(
+                            widget.event.r1Date +
+                                '      ' +
```

Comment:
To the left of the dates there should be a calendar icon . it is not there.Check figma once